### ISSUE TITLE
Refactoring a path with ".." in it causes an issue.

### ISSUE THREAD

**Issue by IngwiePhoenix** [jtangelder/sass-loader#91]
I am currently building a "custom" Bootstrap by going haywire with the config, and creating my own entry SCSS file. Truth be told, I am not familiar with SCSS. It's my first time of use.

I am getting this error:

```
    ERROR in ./~/css-loader?keepSpecialComments=0&processImport=true&rebase=true&relativeTo=/Users/Ingwie/Work/BIRD3&shorthandCompacting=true&target=/Users/Ingwie/Work/BIRD3/cdn/app!./~/sass-loader?includePaths[]=/Users/Ingwie/Work/BIRD3/bower_components/bootstrap-sass/assets/stylesheets&includePaths[]=/Users/Ingwie/Work/BIRD3/bower_components&includePaths[]=/Users/Ingwie/Work/BIRD3/node_modules&includePaths[]=/Users/Ingwie/Work/BIRD3/themes!./web-lib/bootstrapper.scss
    Module not found: Error: Cannot resolve 'file' or 'directory' ../fonts/glyphicons-halflings-regular.ttf in /Users/Ingwie/Work/BIRD3/web-lib
    resolve file
      /Users/Ingwie/Work/BIRD3/fonts/glyphicons-halflings-regular.ttf doesn't exist
      /Users/Ingwie/Work/BIRD3/fonts/glyphicons-halflings-regular.ttf.js doesn't exist
      /Users/Ingwie/Work/BIRD3/fonts/glyphicons-halflings-regular.ttf.json doesn't exist
      /Users/Ingwie/Work/BIRD3/fonts/glyphicons-halflings-regular.ttf.oj doesn't exist
    resolve directory
      /Users/Ingwie/Work/BIRD3/fonts/glyphicons-halflings-regular.ttf/package.json doesn't exist (directory description file)
      /Users/Ingwie/Work/BIRD3/fonts/glyphicons-halflings-regular.ttf doesn't exist (directory default file)
    [/Users/Ingwie/Work/BIRD3/fonts/glyphicons-halflings-regular.ttf]
    [/Users/Ingwie/Work/BIRD3/fonts/glyphicons-halflings-regular.ttf.js]
    [/Users/Ingwie/Work/BIRD3/fonts/glyphicons-halflings-regular.ttf.json]
    [/Users/Ingwie/Work/BIRD3/fonts/glyphicons-halflings-regular.ttf.oj]
     @ ./~/css-loader?keepSpecialComments=0&processImport=true&rebase=true&relativeTo=/Users/Ingwie/Work/BIRD3&shorthandCompacting=true&target=/Users/Ingwie/Work/BIRD3/cdn/app!./~/sass-loader?includePaths[]=/Users/Ingwie/Work/BIRD3/bower_components/bootstrap-sass/assets/stylesheets&includePaths[]=/Users/Ingwie/Work/BIRD3/bower_components&includePaths[]=/Users/Ingwie/Work/BIRD3/node_modules&includePaths[]=/Users/Ingwie/Work/BIRD3/themes!./web-lib/bootstrapper.scss 3:3646-3698
```

In Bootstrap's gylphcons.scss file, they refference the font files with "../glyphs.svg", for instance. Using the plain CSS version let's this work and resolve. But it appears that this module doesn't play nicely with this option.

For now I am commenting the glyphs out (I am using font-awesome too anyway) but would like to know if this could be resolved. Thanks!

**Comment by jhnns** [comment]
All url()s must be relative to the sass main entry file, because the css-loader resolves the url()s based on this location.

[Less](http://lesscss.org/) has an option which rewrites the urls accordingly, but Sass has not afaik. Have you tried overriding the [`$icon-font-path`](https://github.com/twbs/bootstrap-sass/blob/e0ddfcdf288c23bd97b8511a290db7291ccf1860/assets/stylesheets/bootstrap/_variables.scss#L83) instead?

**Comment by jsg2021** [comment]
Something is strange with '..' paths.  I have a `_fonts.scss` as a sibling to my `app.scss` file, which imports that file.  In my fonts file, I import several fonts in a parent-sibling path.

```scss
@import "../fonts/droid-serif/index.css";
@import "../fonts/open-sans/index.css";
...
```
once i work around #98 , i get:

```bash
ERROR in ./~/css-loader!./~/sass-loader?includePaths[]=/Users/jgrimes/Workspace/nti.web.mobile/src/main/resources/vendor/foundation/scss!./src/main/resources/scss/app.scss
Module build failed: 
@import "../fonts/droid-serif/index.css";
       ^
      File to import not found or unreadable: ../fonts/droid-serif/_index.css.scss/Users/jgrimes/Workspace/nti.web.mobile/src/main/resources/scss/
      in /Users/jgrimes/Workspace/nti.web.mobile/src/main/resources/scss/_fonts.scss (line 1, column 9)
Error: 
@import "../fonts/droid-serif/index.css";
       ^
      File to import not found or unreadable: ../fonts/droid-serif/_index.css.scss/Users/jgrimes/Workspace/nti.web.mobile/src/main/resources/scss/
      in /Users/jgrimes/Workspace/nti.web.mobile/src/main/resources/scss/_fonts.scss (line 1, column 9)
    at options.error (/Users/jgrimes/Workspace/nti.web.mobile/node_modules/node-sass/lib/index.js:276:32)
 @ ./src/main/resources/scss/app.scss 4:14-229
```

**Comment by jhnns** [comment]
Oops, that's probably a bug introduce with `1.0.2`. We have to take care of `.css` file extensions

**Comment by skovhus** [comment]
It works in `1.0.0`, but not in `1.0.1` and `1.0.2`.

**Comment by jorrit** [comment]
Does it work in 1.0.3?

**Comment by jorrit** [comment]
Does it work in 1.0.3?

**Comment by jhnns** [comment]
Should be fixed with 2.0.0
